sunkit_magex/pfss/utils.py

```diff
@@ -10,51 +10,8 @@
 from astropy import units as u
 from astropy.wcs import WCS
 
-import sunkit_magex.pfss.map
-
-
-def fix_hmi_meta(hmi_map):
```

Comment:
Should we deprecate this?

Reply:
Why? It isn't needed anymore. The answer is to remove it.